mne/source_estimate.py

```diff
@@ -3194,21 +3195,33 @@ def stc_near_sensors(evoked, trans, subject, distance=0.01, mode='sum',
         zero-order hold. See Notes.
     project : bool
         If True, project the electrodes to the nearest ``'pial`` surface
-        vertex before computing distances.
+        vertex before computing distances. Only used when doing a
+        surface projection.
     %(subjects_dir)s
+    src : instance of SourceSpaces
+        The source space.
+
+        .. warning:: If a surface source space is used, make sure that
+                     ``surf='pial'`` was used during construction.
     %(verbose)s
 
     Returns
     -------
     stc : instance of SourceEstimate
-        The surface source estimate.
+        The surface source estimate. If src is None, a surface source
+        estimate will be produced, and the number of vertices will equal
+        the number of pial-surface vertices that were close enough to
+        the sensors to take on a non-zero volue. If src is not None,
+        a surface, volume, or mixed source estimate will be produced
+        (depending on the kind of source space passed) and the
+        vertices will match those of src (i.e., there may be me
+        many all-zero values in stc.data).
 
     Notes
     -----
-    This function projects the ECoG sensors to the pial surface (if
-    ``project``), then the activation at each pial surface vertex is given
-    by the mode:
-
+    For surface projections, this function projects the ECoG sensors to
+    the pial surface (if ``project``), then the activation at each pial
+    surface vertex is given by the mode:
```

Comment:
You probably need to restore the newline here for rst to render properly. The error you saw was because of an extra space indentation that I pushed a fix for